docs/install.md

```diff
@@ -33,36 +34,48 @@ Drupal Compatibility
     <td> Drush 10 </td>
     <td> master </td>
     <td> 7.1+ </td>
+    <!-- Released Oct 2019 -->
+    <td> <i>tbd</i> </td>
```

Comment:
```suggestion
    <td> <i>TBD</i> </td>
```

Reply:
I think it is contextually clear that the EOL date for the most recent version is not set yet. Uppercasing TBD is fine though.